source/docs/software/pathplanning/system-identification/viewing-diagnostics.rst

```diff
@@ -1,53 +1,63 @@
 Viewing Diagnostics
 ===================
 
-The first diagnostic to look at is the r-squared - it should be somewhere north of ~.9. If it is significantly lower than this, there is likely a problem with your identification data.
+Goodness-of-Fit Metrics
+-----------------------
+
+There are three numerical accuracy metrics that are computed with this tool: acceleration r-squared, simulated velocity r-squared, and RMSE.
+
+.. image:: images/viewing-metrics.png
+   :alt: Analysis accuracy metrics
+
+The `r-squared <https://en.wikipedia.org/wiki/Coefficient_of_determination>`__ metric refers to the goodness of fit of the tool's `OLS (Ordinary Least Squares) fit <https://en.wikipedia.org/wiki/Ordinary_least_squares>`__.  The `RMSE (Root Mean Squared Error) <https://en.wikipedia.org/wiki/Root-mean-square_deviation>`__ is the error (in the measured units) between the collected velocities and the predicted velocities from a simulation based on the feedforward gains.
 
-The next diagnostic to look at is the RMSE to make sure that the error is reasonable and within your desired constraints.
+The acceleration r-squared is the fraction of the variance in measured acceleration (used as the independent variable in the SysId regression) explained by the linear model.  This is usually quite low, because acceleration is very susceptible to system noise.
```

Comment:
Maybe we could provide a reference to how low is "quite low".  I don't think most people reading this have a frame of reference here.